### ISSUE TITLE
Normal estimation using gradient of density

### ISSUE THREAD

**Issue by mra-h** [kakaobrain/nerf-factory#44]
Hello,

Thank you so much for this implementation of ref nerf!

I have a question regarding on the gradient of the density to compute the normals. Why is `raw_density.sum()` used as outputs in torch.autograd as shown [here](https://github.com/kakaobrain/nerf-factory/blob/31d74696afc89c23609e81d20c7c2eabc3f1c928/src/model/refnerf/model.py#L153).

Is it different than calculating the gradient as 
`torch.autograd.grad(raw_density, means, grad_outputs=torch.ones_like(raw_density), retain_graph=True)`. ?
